src/components/cv/andre-godkjenninger.tsx

```diff
@@ -0,0 +1,34 @@
+import { ArenaPerson } from '../../data/api/datatyper/arenaperson';
+import CvInfo from '../felles/cvinfo';
+import { formaterDato } from '../../utils/formater';
+import { safeMap } from '../../utils';
+import { FolderPlusIcon } from '@navikt/aksel-icons';
+import { BodyShort, Label } from '@navikt/ds-react';
+import EMDASH from '../../utils/emdash';
+
+const AndreGodkjenninger = (props: Pick<ArenaPerson, 'andreGodkjenninger'>) => {
+    const { andreGodkjenninger } = props;
+    const annenGodkjenningListe = safeMap(andreGodkjenninger, (annenGodkjenning, index) => (
+        <div key={`andregodkjenninger-${index}`} className="underinformasjon">
+            <Label size="small" as="p" key={`andregodkjenninger-${index}`}>
```

Comment:
Går nok fint å bruke index her, men bare så man vet det til senere:
Man burde generelt bruke mer "faste" keys over index, dette er fordi React bruker keys for å se om noe har endret seg i domen. I dette tilfelle vil jeg tro at rekkefølgen på andregodkjenninger vil forbli det samme, men bare sånn til info :D https://legacy.reactjs.org/docs/lists-and-keys.html#:~:text=We%20don't%20recommend%20using,cause%20issues%20with%20component%20state.